indexer-app/resources/generics/grid/v0.0.1/index.json

```diff
@@ -0,0 +1,84 @@
+{
+  "MetadataSpecification": {
+      "URL": "https://cdn.earthdata.nasa.gov/generic/index/v0.0.1",
+      "Name": "Generic-Index",
+      "Version": "0.0.1"
+  },
+  "SubConceptType": "GRD",
+  "Indexes":
+  [
+    {
+      "Description": "Long Name",
+      "Field": ".LongName",
+      "Name": "Long-Name",
+      "Mapping": "string"
+    },
+    {
+      "Description": "Type - maybe change",
+      "Field": ".GridDefinition.CoordinateReferenceSystemID.Type",
+      "Name": "CoordinateReferenceSystemID-Type",
+      "Mapping": "string"
+    },
+    {
+      "Description": "URL - maybe changee",
+      "Field": ".GridDefinition.CoordinateReferenceSystemID.URL",
+      "Name": "CoordinateReferenceSystemID-URL",
+      "Mapping": "string"
+    },
+    {
+      "Description": "Reference System",
+      "Field": ".GridDefinition.CoordinateReferenceSystemID",
+      "Name": "Grid-System",
+      "Mapping": "string",
+      "Indexer": "complex-field",
+      "Configuration": {"sub-fields": ["Type", "URL"]}
+    },
+    {
+      "Description": "Mock index for property field",
+      "Field": ".Name",
+      "Name": "Name",
+      "Type": "graph",
+      "Indexer": "property"
+  },
+  {
+      "Description": "Mock index for property field",
```

Comment:
Are the duplicates of descriptions intentional?

Reply:
no, these have been updated.